intel-retail/automated-self-checkout
# Multi cam

## PR Checklist



## What are you changing?

Adding multiple camera simulators for different streams.
Add multiple output streams for better representation.

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- make run-pipeline-server
- open localhost:3000 to see grafana dashboard
- See all 3 pipeline FPS averages and output streams

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
